Mathlib/Topology/Algebra/TopologicallyNilpotent.lean

```diff
@@ -125,6 +125,13 @@ theorem add {a b : R} (ha : IsTopologicallyNilpotent a) (hb : IsTopologicallyNil
     IsTopologicallyNilpotent (a + b) :=
   ha.add_of_commute hb (Commute.all ..)
 
+/-- The topological nilradical of a ring with a linear topology -/
+def topologicalNilradical : Ideal R where
```

Comment:
Can you add at least the obvious `...mem_iff` lemma?

Reply:
I did that and also moved `topologicalNilradical` to the `_root_` namespace.